viahtml/templates/banner.html

```diff
@@ -88,18 +88,21 @@
     function setupExternalLinkHandler(mode) {
       if (mode === "new-tab") {
         document.addEventListener("click", function (event) {
-          if (!event.target.closest) {
-            // Do nothing in browsers that don't support `element.closest` (IE 11).
```

Comment:
The Hypothesis client stopped supporting IE 11 several years ago, so we don't need a fallback here.